src/circleci/rollcage/core.clj

```diff
@@ -187,3 +188,24 @@
 (def warning  (partial notify "warning"))
 (def info     (partial notify "info"))
 (def debug    (partial notify "debug"))
+
+(defn rollbar-reporter
+  "Returns a rollbar reporter function that takes an exception and
+  reports it to rollbar with the given params."
+  [rollcage-client params]
+  (infof "rollbar reporting configured for %s" params)
+  (fn [^Throwable ex]
+    (try
+      (error rollcage-client ex {:params params})
+      (catch Exception e
+        (errorf e "Unable to report exception to rollbar, original exception follows")
+        (.printStackTrace ex)))))
+
+(defn configure-rollbar!
+  "Make rollcage handle uncaught exceptions and return a rollcage client."
+  [rollbar-token environment]
```

Comment:
I don't think this function adds value to Rollcage - I'd rather see these lines of code explicitly in the `main` function of each project. I could be convinced otherwise.

One thing I don't like is that the call can only specify one parameter `environment` - if the user wanted to specify `:code-version` too,  then we would have to change this function. It would be better to take all client options as the second param.

But I think this function belongs explicitly in each app tbh.